diff/src/main/java/com/adaptivescale/rosetta/diff/kinetica/KineticaTester.java

```diff
@@ -402,4 +397,23 @@ private Optional<Table> getTable(String tableName, Database targetValue) {
     private Optional<View> getView(String viewName, Database targetValue) {
         return targetValue.getViews().stream().filter(targetView -> targetView.getName().equals(viewName)).findFirst();
     }
+
+    private static boolean areColumnPropertiesEqual(List<ColumnProperties> list1, List<ColumnProperties> list2) {
```

Comment:
maybe instead of `list1`, `list2` we can use something like local..., target...

Reply:
i changed it to listLocal and listTarget